File: src/net/eduard/permissions/command/PermissionsUserCommand.java

package net.eduard.permissions.command;

import net.eduard.api.lib.manager.CommandManager;

public class PermissionsUserCommand extends CommandManager {

	public PermissionsUserCommand() {
		super("user", "usuario");
		setDescription("Controlar as permissões e grupos dos jogadores");
		register(new PermissionsUserAddGroupCommand());
		register(new PermissionsUserRemoveGroupCommand());
		register(new PermissionsUserAddPermissionCommand());
		register(new PermissionsUserRemovePermissionCommand());
		register(new PermissionsUserHelpCommand());
		register(new PermissionsUserSetPrefixCommand());
		register(new PermissionsUserSetSuffixCommand());

	}

}

File: src/main/java/net/eduard/permissions/command/PermissionsUserCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.command.Command


class PermissionsUserCommand : Command("user", "usuario") {
    init {
        description = "Controlar as permissões e grupos dos jogadores"
        register(PermissionsUserAddGroupCommand())
        register(PermissionsUserRemoveGroupCommand())
        register(PermissionsUserAddPermissionCommand())
        register(PermissionsUserRemovePermissionCommand())
        register(PermissionsUserHelpCommand())
        register(PermissionsUserSetPrefixCommand())
        register(PermissionsUserSetSuffixCommand())
    }
}